src/test/java/uk/gov/hmcts/reform/professionalapi/service/ProfessionalUserServiceTest.java

```diff
@@ -67,12 +66,12 @@ public void retrieveUserByEmail() {
         assertEquals(professionalUser.getEmailAddress(), user.getEmailAddress());
     }
 
-    @Test(expected = HTTPException.class)
+    @Test//(expected = HTTPException.class)
```

Comment:
Why is this commented out. If we are NOT expecting and exception to be thrown then please just delete the expected annotation.

Reply:
done